pkg/controller/sdk/status_accessor.go

```diff
@@ -73,3 +73,56 @@ func (r *reflectedStatusAccessor) SetStatus(status interface{}) {
 		r.status.Set(reflect.ValueOf(status))
 	}
 }
+
+// FinalizersAccessor is the interface for a Resource that implements the getter and setting for
+// accessing its Finalizer set.
+// +k8s:deepcopy-gen=true
+type FinalizersAccessor interface {
```

Comment:
This should move to it's own go file. Perhaps `finalizer_accessor.go`

Reply:
Done.